source-gladly/source_gladly/api.py

```diff
@@ -60,6 +60,15 @@ async def fetch_events(
                 f"not updating cursor since last_ts ({last_ts}) <= log_cursor ({log_cursor})"
             )
             return
+        elif last_ts - log_cursor < timedelta(hours=6):
```

Comment:
nit: i suspect this control flow could be cleaned up further, but this is correct :shrug: